server/routes/inPrisonEducationRecordsResolver.ts

```diff
@@ -33,4 +33,31 @@ const mostRecentCompletedInPrisonEducationRecords = (
   }
 }
 
-export { completedInPrisonEducationRecords, mostRecentCompletedInPrisonEducationRecords }
+/**
+ * Returns an InPrisonEducationRecords object where the eduction records are filtered so that only the completed courses
+ * within the last 12 months are returned
+ */
+
+const completedInPrisonEducationRecordsWithinLast12Months = (
+  allInPrisonEducationRecords: InPrisonEducationRecords,
+): InPrisonEducationRecords => {
+  return {
+    ...allInPrisonEducationRecords,
+    educationRecords: (allInPrisonEducationRecords.educationRecords || [])
+      .filter(inPrisonEducation => inPrisonEducation.courseCompleted === true)
```

Comment:
I'm not sure this is the right place to do this "status" filtering? 🤔  It _might_ be, but I'm not sure .. I'm thinking it through with a thought to the linked stories where we need to show In Progress and Withdrawn courses .... 🤔 

What is the `InPrisonEducationRecords` object? Where is it set? And crucially, what sets the `courseCompleted` property ?